adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationContext.cs

```diff
@@ -108,15 +108,15 @@ public AuthenticationContext(string authority, bool validateAuthority, TokenCach
         {
         }
 
-        internal AuthenticationContext(IHttpManager httpManager, string authority, AuthorityValidationType validateAuthority,
+        internal AuthenticationContext(IServiceBundle serviceBundle, string authority, AuthorityValidationType validateAuthority,
             TokenCache tokenCache)
         {
             // If authorityType is not provided (via first constructor), we validate by default (except for ASG and Office tenants).
             this.Authenticator = new Authenticator(authority, (validateAuthority != AuthorityValidationType.False));
             this.TokenCache = tokenCache;
 
-            _httpManager = httpManager ?? new HttpManager();
-            _wsTrustWebRequestManager = new WsTrustWebRequestManager(_httpManager);
+            _serviceBundle = serviceBundle ?? ServiceBundle.CreateForProduction();
+            _wsTrustWebRequestManager = new WsTrustWebRequestManager(_serviceBundle);
```

Comment:
>_wsTrustWebRequestManager = new WsTrustWebRequestManager(_serviceBundle); [](start = 12, length = 73)

would it make sense to put this in the ServiceBundle as well even though it will be kind of a fun "dependency"?

Reply:
done